projects/packages/forms/src/blocks/contact-form/child-blocks.js

```diff
@@ -26,6 +26,10 @@ const FieldDefaults = {
 	supports: {
 		reusable: false,
 		html: false,
+		// This doesn't opt in to support for any dimensions controls,
+		// but it's required to ensure the 'Styles' tab renders in the inspector.
+		// This is probably a bug in WordPress core.
```

Comment:
> This is probably a bug in WordPress core.

Worth logging issue in Gutenberg repo?

Reply:
I would think making issues counts as a contribution (I'd get props for it when the issue is solved). 🤷‍♂️

Reply:
Logged here - https://github.com/WordPress/gutenberg/issues/69404

Reply:
Also pushed a commit that links to the issue in the code comment.